src/components/Header.css

```
@@ -0,0 +1,46 @@
+.header-block {
+  padding: 1.6em 0.9em;
+}
+
+.block {
+  margin: 0 0.9em;
+  box-sizing: border-box;
+  display: inline-block;
+  height: 6.5em;
+  width: 17em;
+  background: #4c56c0;
+  color: #ffffff;
+  border-top-right-radius: 1em;
+  border-bottom-right-radius: 1em;
+  border-top-left-radius: 1em;
+  border-bottom-left-radius: 1em;
```

Так як значення однакові, можна об'єднати 4 в 1: 
```suggestion
  border-radius: 1em;
```